docs/source/plugins/optional/varianter_yaml_to_mux.rst

```diff
@@ -140,8 +140,8 @@ The environment created for the nodes ``fedora`` and ``osx`` are:
 
 Note that due to different usage of key and values in environment we disabled
 the automatic value conversion for keys while keeping it enabled for values.
-This means that the value can be of any YAML supported value, eg. bool, None,
-list or custom type, while the key is always string.
+This means that the key is always string and the value can be of any YAML supported value, eg. bool, None,
```

Comment:
"the key is always *a* string"

This also seems to not follow the line breaking from the original document.

Reply:
Expecting None to be removed from here as there is no way it can be provided. You could add that as a limitation explicitly in the next sentence. Something like
`Please be aware that the None type can be passed to parameters only as a string`